plugins/parcelport/verbs/unordered_map.hpp

```diff
@@ -0,0 +1,354 @@
+////////////////////////////////////////////////////////////////////////////////
+//  Copyright (c) 2016 John Biddiscombe
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+////////////////////////////////////////////////////////////////////////////////
+
```

Comment:
I trust this will move to the directory where the other concurrent containers live (will live).